app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -1,7 +1,9 @@
 <%
+  hide_feedback_form ||= false
```

Comment:
Could this be named `omit_feedback_form` to keep it consistent with `omit_header`?

Reply:
Good point!